web/src/app/pages/tasks/viewtasks.controller.js

```diff
@@ -71,20 +65,10 @@
             });
         };
 
-        var getUserCurrenTasks = function() {
-            $http.post(IG.api + 'tasks/getUserTask', userID).then(function(response) {
-                if (response.data.status == "success") {
-                    $scope.list4 = response.data.data.Item.task;
-                    print("Tasks retrieved");
-                } else if (response.data.status == "error") {
-                    print("User authentication error");
-                }
-            });
-        };
-        getUserCurrenTasks();
+        $scope.list4 = userTask.task;
 
         $scope.hideMe = function() {
-            if ($scope.list4 != undefined) return $scope.list4.length > 0;
+            if ($scope.list4) return $scope.list4.length > 0;
```

Comment:
No need of checking for `length > 0`. You can even say `$scope.list4 && $scope.list4.length`